netty3/src/play/server/netty3/PlayHandler.java

```diff
@@ -524,7 +524,7 @@ Request parseRequest(HttpRequest nettyRequest, MessageEvent messageEvent)
             getRemoteIPAddress(messageEvent),
             nettyRequest.getMethod().getName(),
             uri.getPath(),
```

Comment:
That `uri.getPath()` is an already decoded path. Shouldn't we add a test + fix (`uri.getRawPath()`) for that too? :upside_down_face: 

Back in the old code (before commit: a9c68f127dd64cc95c39e0b42795950556ab37fd) that `path` and `querystring` was just the split from `nettyRequest.getUri()`: [netty3/src/play/server/netty3/PlayHandler.java@8c6fbc0e0fea](https://github.com/replay-framework/replay/blob/8c6fbc0e0fea36ee434e0a4514dcf9bfb980b762/netty3/src/play/server/netty3/PlayHandler.java#L498C1-L504C10)

Reply:
@xabolcs Good point. Added tests for `getRawPath()` as well: https://github.com/replay-framework/replay/pull/649/commits/716b5c290e7b2937ce608e5fbcdd1cdbe676d7e7